src/components/clusterWizard/wizardTransition.ts

```diff
@@ -167,14 +167,17 @@ export const getWizardStepHostStatus = (
 ): Host['status'] => {
   const { status } = host;
   if (['insufficient', 'pending-for-input'].includes(host.status)) {
```

Comment:
To be consistent with the other change:
```suggestion
  if (['insufficient', 'pending-for-input'].includes(status)) {
```

Reply:
done within #472